lib/resolve-repositories.js

```diff
@@ -51,7 +51,14 @@ export async function resolveRepositories(state, repositories) {
         per_page: 100,
       }
     )) {
-      resolvedRepositories.push(...response.data);
+      let selectedRepositories = response.data;
+
+      if (repoPattern !== '*') {
+        const repoRegex = new RegExp(`/${repoPattern}/`)
```

Comment:
Approach looks good, but I think you we need this

```suggestion
        const repoRegex = new RegExp(`^${repoPattern.replace(/\*/g, '.*')}$`)
```

Might be a good time to introduce some tests ;) I like https://github.com/lukeed/uvu. I would start by just creating a unit test in `test/unit/resolve-repositories.test.js`

We can also ship this pull request once you confirmed that it's working, then do a follow up PR for adding tests

Reply:
> Approach looks good, but I think we need this
> 
> Might be a good time to introduce some tests ;) I like [lukeed/uvu](https://github.com/lukeed/uvu). I would start by just creating a unit test in `test/unit/resolve-repositories.test.js`

Yes, we needed it. Nice catch
 
> We can also ship this pull request once you confirmed that it's working, then do a follow up PR for adding tests

Working on the tests right now. :)